tests/unit/test_completion.py

```diff
@@ -3,7 +3,7 @@
 Unit tests for linodecli.completion
 """
 
-from unittest.mock import mock_open, patch
+pass
```

Comment:
Let's just remove it?

Reply:
Oops, no clue how that got there lol. Let me fix that up